bdbstore/src/main/java/org/apache/qpid/server/store/berkeleydb/CoalescingCommiter.java

```diff
@@ -134,18 +134,21 @@ public void abort(RuntimeException databaseException)
     private static class CommitThread extends Thread
     {
         private static final Logger LOGGER = LoggerFactory.getLogger(CommitThread.class);
-        private static final int JOB_QUEUE_NOTIFY_THRESHOLD = 8;
 
+        private final int jobQueueNotifyThreshold;
+        private final long commiterNotifyTimeout;
         private final AtomicBoolean _stopped = new AtomicBoolean(false);
         private final Queue<CommitThreadJob> _jobQueue = new ConcurrentLinkedQueue<>();
         private final Object _lock = new Object();
         private final EnvironmentFacade _environmentFacade;
 
         private final List<CommitThreadJob> _inProcessJobs = new ArrayList<>(256);
 
-        public CommitThread(String name, EnvironmentFacade environmentFacade)
+        public CommitThread(String name, int commiterNotifyThreshold, long commiterNotifyTimeout, EnvironmentFacade environmentFacade)
```

Comment:
I think that a better name for `commiterNotifyTimeout` could be `commiterWaitTimeout` as it would convey the purpose of timeout more accurately